nix/overlays/required.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, enableProfiling }:
+
+with import ../../lib.nix;
+
+with pkgs.haskell.lib;
+
+let
+  addRealTimeTestLogs = drv: overrideCabal drv (attrs: {
+    testTarget = "--show-details=streaming";
+  });
+in
+
+self: super: {
+
+    ########################################################################
+    # Overides of Cardano SL packages
+
+    ########################################################################
+    language-plutus-core = addRealTimeTestLogs super.language-plutus-core;
+    # The base Haskell package builder
+
+    mkDerivation = args: super.mkDerivation (args // {
```

Comment:
I'm really not a fan of overriding `mkDerivation`. Are we sure this isn't going to mess with other things unexpectedly? Shouldn't there be a way to override just the haskell builder attrs?

Reply:
that's the only way to apply a single override to the entire tree at once. @cleverca22 was the one that implemented it, and it's a common pattern to accommodate that goal. cardano-sl has been using the overlay mkDerivation work for a couple of releases now.